spec/index.html

```diff
@@ -319,50 +319,87 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
     <h3>Triple Terms and Reification</h3>
 
     <p>A <a>triple term</a> is an [=RDF triple=] used as an [=RDF term=]
-      in another triple. This use is a reference to a <a>proposition</a>. For
-      the triple term to also be asserted, it must [=appear=] in the same graph as both an <a>asserted
-      triple</a> and an unasserted <a>triple term</a>. This allows for statements to be made about statements
-      independent of their assertion within an <a>RDF graph</a>.
-    </p>
-
-    <p>[=RDF terms=] that [=appear=] in a [=triple term=] have the same
-      <a data-lt="denotes">denotation</a> as when they appear in an
-      [=asserted triple=] in the <a data-lt="RDF graph">graph</a>.
-      For this reason, we say that triple terms are
-      <dfn class="lint-ignore">transparent</dfn>.</p>
+      in another triple. 
+      
+<!-- When read literally, one could understand this to say that a triple term _references_ (whatever references may mean here) a proposition. But, I think it is simply meant that a triple term encodes a proposition, just like an RDF triple (?) If so, then perhaps the ambiguity could be removed by the following. 
+-->
+<!-- original:
+      This use is a reference to a <a>proposition</a>. 
+-->
+As an [=RDF triple=], it encodes a <a>proposition</a>.
```

Comment:
While we do say "encode" in [1.8](https://www.w3.org/TR/rdf12-concepts/#entailment) too, we actually say "The denotation of a triple is a proposition," [in Semantics](https://www.w3.org/TR/rdf12-semantics/#simple_entailment_properties). I think we should instead correct "encode" to "denote" in all places in Concepts.

Reply:
Indeed, the same terminology should be used consistently everywhere.